nansat/vrt.py

```diff
@@ -1729,7 +1729,8 @@ def _make_source_bands_xml(src_in):
 
         # find DataType of source (if not given in src)
         if src['SourceBand'] > 0 and 'DataType' not in src:
-            raster_band = gdal.Open(src['SourceFilename']).GetRasterBand(src['SourceBand'])
+            ds = gdal.Open(src['SourceFilename'])
```

Comment:
Would you mind adding a comment so we don't revert this change later, having forgotten what it was for?
For example:
```suggestion
            # prevent Python from releasing the dataset's handle
            # prematurely
            ds = gdal.Open(src['SourceFilename'])
```

Reply:
Yes, of course